bin/spack

```diff
@@ -208,7 +208,7 @@ def main(args):
 
     if args.profile:
         import cProfile
-        cProfile.runctx('_main(args, unknown)', globals(), locals())
+        cProfile.runctx('_main(args, unknown)', globals(), locals(), sort='cumtime')
```

Comment:
This may not work in Python 2.6. I've personally removed the sort arg for this function call in 2 separate PRs.